images/miq-app/docker-assets/container-deploy-common.sh

```diff
@@ -67,6 +70,22 @@ fi
 
 }
 
+function check_version_gt() { 
+# Check if upgrade version is actually greater than stored PV version
+# -V sorts alphanumeric versions within text, will always return oldest version first
+# Compare sort version result against upgrade version
+
```

Comment:
In second thought this is so far fetched so lets just leave it as is.
(in order to make it happen one must either edit the template or build a new image...)